src/redux/books/books.js

```diff
@@ -0,0 +1,41 @@
+import { v4 as uuidv4 } from 'uuid';
+// Actions
+
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/ADD_BOOK';
```

Comment:
-  Kindly correct this error. `'bookstore/books/ADD_BOOK';`. The correct path should be `'bookstore/books/REMMOVE_BOOK'`;

Reply:
Yeah, thanks